tests/checks/PrismaClientOnContext.test.ts

```diff
@@ -1,17 +1,18 @@
 import gql from 'graphql-tag'
 import { objectType, queryType } from 'nexus'
 import { testIntegration, TestIntegrationParams, testIntegrationPartial } from '../__helpers__/testers'
+import { clearKleurFormat } from '../../src/lib/messenger/Messenger'
 
 const base = testIntegrationPartial({
   database: `
     model User {
       id        String   @id
-			profile   Profile? @relation(fields: [profileId], references: [id])
-			profileId String
+			profile   Profile @relation(fields: [profileId], references: [id])
+			profileId String   @unique
     }
 		model Profile {
       id    String  @id
-			user  User    @relation
+			user  User?    @relation
```

Comment:
What triggered the need for these changes?

Reply:
In the last version of prisma `3.12`, they required the relation key `profileId` to be in the required field.
@unique added by prisma format

Reply:
Ok, given that change from Prisma, to keep this project simple, we should probably do as you've done in this PR, raise the requires Prisma version to `3.12`.

I'm open to not doing that if you want to taken on the complexity and tests of cleanly supporting the past versions.